### ISSUE TITLE
create systemd service for ecsstandalone

### ISSUE THREAD

**Issue by bottkars** [EMCECS/ECS-CommunityEdition#155]
- [ ] create a systemd file " /etc/systemd/System/docker-ecsstandalone.service
  "from step1_ecs_singlenode_install.py
- [ ] enable and start the Container from systemd

```
[Unit]
Description=EMC ECS Standalone Container
Requires=docker.service
After=docker.service

[Service]
Restart=always
ExecStart=/usr/bin/docker start -a ecsstandalone
ExecStop=/usr/bin/docker stop -t 2 ecsstandalone

[Install]
WantedBy=default.target

```

execute

```
systemctl daemon-reload
systemctl enable docker-ecsstandalone
systemctl start docker-ecsstandalone
```

Example:
![image](https://cloud.githubusercontent.com/assets/8255007/17587104/5d2a4ebc-5fc6-11e6-857a-aecaf7786914.png)

**Comment by padthaitofuhot** [comment]
@bottkars You'll be pleased to know systemd support has been included in #201

**Comment by bottkars** [comment]
Awesome !!!!
